### PR TITLE
Meta tags for SEO

### LINKED ISSUE #455: Add meta data info to pages
**Parent Issue #311**

* Add metadata info to the pages for Search engines to be able to easily crawl the page

### PR DESCRIPTION
**Pull request by hemantjadon**
* Adding`meta` tags for SEO

* Please suggest some other keywords which can be added.

**Closes #455**

### CODE REVIEW

**Comment by hemantjadon** [comment]
@singhpratyush @vibhcool @djmgit @kavithaenair @Achint08 
Please suggest other tags which might be added.
And alternate description if necessary

**Comment by hemantjadon** [comment]
@mariobehling I read further regarding the `meta` tags but it seems not much meta tag are really important for SEO. [Here](https://developers.google.com/search/docs/guides/enhance-site#add-a-sitelinks-searchbox-for-your-site) is the list of all which are understood by google

The only important bit here is the `<meta description=" ... " />` tab which we require.  That I have added I can add other tags also but they are not important. Please verify the description tag that it is relevent. And specify alternatives.

**Comment by hemantjadon** [comment]
@Achint08 the other tags can be added but they don't contribute to SEO as the link I mentioned in previous comment states.

**Comment by Achint08** [comment]
@hemantjadon I totally understand that. 
I think the description can be altered a bit. @mariobehling can guide you with this. 
Basically, description should also state that it is a search engine. 
You can search for google, duckduckgo, Yahoo, loklak on google and take a reference to make the desciption more clear.
Kindly see if this applies. :)

**Comment by Achint08** [comment]
@hemantjadon One thing I would like you to add up is the `og` open graph tags (for facebook) and `twitter` tags (for twitter). This will help to create a thumbnail whenever user mentions the link.

Currently, it looks like this.
screen shot 2017-07-15 at 9 33 53 pm

How the thumbnail should look like. 
screen shot 2017-07-15 at 9 36 35 pm

You can take duckduckgo tags for reference. How they use the tags.

**Comment by hemantjadon** [comment]
All suggestions are welcome regarding this.
@mariobehling Your suggestions regarding this?

**Comment by hemantjadon** [comment]
@Achint08 those tags can be taken up in another PR

**Comment by Achint08** [comment]
@hemantjadon Sure. :)

File: src/index.html [hunk 1]
```diff
@@ -7,9 +7,18 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
 	<link rel="icon" type="image/png" sizes="32x32" href="assets/images/favicons/favicon.png">
-	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
-	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.1.3/Chart.bundle.min.js"></script>
+
+	<!-- Meta Tags for SEO -->
+	<meta name="keywords" content="loklak search open data source knowledge social media">
```
Comment (vibhcool) [inline src/index.html]: are `meta` keywords needed? they aren't used by search engines.

**Comment by vibhcool** [comment]
see, how this looks as description content -> `Search on social media using Loklak Search. for search use #hashtag ;  "search text" for exact matches;  near:location for messages near a location;  @user finds user mentions;  from:user shows messages of a user; find followers using followers:user `

File: src/index.html (hunk 1 above)
Comment (hemantjadon) [inline src/index.html]: @vibhcool only meta description and robots  are important, and default for robots is `<meta name="robots" content="index,follow">` that's why no need to specify this explicitly

**Comment by hemantjadon** [comment]
@vibhcool this seems better, What do you think @Achint08 @singhpratyush?

File: src/index.html (hunk 1 above)
Comment (vibhcool) [inline src/index.html]: @hemantjadon and shouldn't the keywords be separated by comma?

Comment (hemantjadon) [inline src/index.html]: There is no need to do so keywords are space separated. Also they are not important. I will remove them as they are not used by any search engine.

**Comment by singhpratyush** [comment]
> `Search on social media using Loklak Search.`

:+1: 

After this, we can mention about peer to peer and open source stuff.

**Comment by hemantjadon** [comment]
@vibhcool @Achint08 @singhpratyush Please review again 😄

**Comment by vibhcool** [comment]
@hemantjadon It looks good 👍 , but IMHO, description shall explain about what it does and why user shall open that link

**Comment by hemantjadon** [comment]
@vibhcool Actually the length of the description string must be at max 150 characters. And right now this string itself is at 160. The Google show ellipses after that we can add information but the basic information should be of 150 characters. I will add the other details after that. but the first sentence should ideally be max 150 chars

**Comment by vibhcool** [comment]
yep, but it isn't much useful description. Approving it :D

**Pull request merged**